instrumentation/kafka/kafka-clients/README.md

```diff
@@ -0,0 +1,6 @@
+# Settings for the Kafka client instrumentation
+
+| System property | Type | Default | Description |
+|---|---|---|---|
+| `otel.instrumentation.kafka.experimental-span-attributes` | Boolean | `false` | Enable the capture of span attributes (experimental). |
+| `otel.instrumentation.kafka.client-propagation.enabled` | Boolean | `true` | Enables client propagation. |
```

Comment:
@mateuszrzeszutek Can you provide more explanation for this functionality?

Reply:
+1

Reply:
How about
```suggestion
| `otel.instrumentation.kafka.client-propagation.enabled` | Boolean | `true` | Enables remote context propagation via Kafka message headers. |
```